### DISCUSSION TITLE
Whitelist mode for "Ad blocking" ?

### DISCUSSION THREAD

**Discussion by Aeris1One** [RaspAP/raspap-webgui#930] [Q&A] [answered]
Hi,
I'm making a presentation and want to setup a WiFi hotspot to let people access a single online website (my Kahoot! clone actually) but I don't want to let people access the internet through the hotspot, only the single website at a specific domain I own (and maybe my Matomo instance too).

So I would like to use the Ad blocking system to block all website except mine...
Before installing RaspAP, can anyone tell me if there's such a whitelist system ? 

Or maybe another way using actual system to only allow access to aeris-one.fr subdomains ?

**Comment by billz** [comment, marked as answer]
Discussed here https://github.com/RaspAP/raspap-webgui/discussions/744

**Reply by Aeris1One** [reply]
Oh ! I didn't see that one 😅
